app/[locale]/hostels/[url_name]/page.tsx

```diff
@@ -0,0 +1,181 @@
+import Link from 'next/link';
+import { Fragment } from 'react';
+import { notFound } from 'next/navigation';
+
+import Heading from '~/components/heading';
+import ImageHeader from '~/components/image-header';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
+import { db, hostels } from '~/server/db';
+
+export async function generateStaticParams() {
```

Comment:
![image](https://github.com/user-attachments/assets/def81db6-2d0d-4569-86bf-f013d3de7979)
should I change them to dynamic 
@GetPsyched

Reply:
Why does that error show up? Any reason?

Reply:
that's what i'm unsure about, other pages use same fetch pattern and dont seem to get this error